### PR TITLE
Remove trusted repayments functionality from the market contract

### PR DESCRIPTION
**Pull request by igorsenych-cw**

### CODE REVIEW

**Review by yuriicw — APPROVED** [review]

File: contracts/Cloudwalk/TransientBorrowRateModel.sol [hunk 1]
```diff
@@ -150,21 +149,6 @@ contract TransientBorrowRateModel is InterestRateModel {
         owner = newOwner;
     }
 
-    /**
-     * @dev Transfers ownership of the contract to a new account (`newOwner`)
-     * Can only be called by the current owner
-     */
-    function updateBlocksPerYear(uint256 newBlocksPerYear)
```
Comment (EvgeniiZaitsevCW) [inline contracts/Cloudwalk/TransientBorrowRateModel.sol:157]: After removing this function the following constant should be removed too:
* `blocksPerYearMax`
* `blocksPerYearMin`

File: contracts/Cloudwalk/ErrorReporter.sol [hunk 2]
```diff
@@ -133,7 +133,6 @@ contract TokenErrorReporter {
     error SetTrustedSupplierOwnerCheck();
     error SetTrustedBorrowerOwnerCheck();
 
-    error TrustedAdminAccountCheck();
     error TrustedBorrowerAccountCheck();
     error TrustedSupplierAccountCheck();
```
Comment (EvgeniiZaitsevCW) [inline contracts/Cloudwalk/ErrorReporter.sol:137]: Looks like the `TrustedSupplierAccountCheck` error is not used anywhere. Do we need it?

**Review by EvgeniiZaitsevCW — APPROVED** [review]
Some minor remarks about unused code entities.

File: contracts/Cloudwalk/TransientBorrowRateModel.sol (hunk 1 above)
Comment (igorsenych-cw) [reply contracts/Cloudwalk/TransientBorrowRateModel.sol]: We will remove `TransientBorrowRateModel` contract in the future. Changes reverted.

File: contracts/Cloudwalk/ErrorReporter.sol (hunk 2 above)
Comment (igorsenych-cw) [reply contracts/Cloudwalk/ErrorReporter.sol:137]: Actually, this error has been removed, no?

Comment (EvgeniiZaitsevCW) [reply contracts/Cloudwalk/ErrorReporter.sol:137]: No :)
It is here.
The `TrustedAdminAccountCheck` error was removed.

**Review by arthur-cw — APPROVED** [review]

**Pull request merged**